janusgraph-backend-testutils/src/main/java/org/janusgraph/olap/OLAPTest.java

```diff
@@ -139,6 +141,37 @@ private int generateRandomGraph(int numV) {
         return numE;
     }
 
+    @Test
+    public void scannerShouldSeeAllVertices() throws Exception {
+        GraphTraversalSource g = graph.traversal();
+        Vertex v1 = g.addV().next();
+        Vertex v2 = g.addV().next();
+        g.V(v1).addE("connect").to(v2).iterate();
+        g.addV().addV().addV().property("p", "v").iterate();
+        g.tx().commit();
+
+        AtomicInteger vertexNum = new AtomicInteger();
+        executeScanJob(new VertexScanJob() {
+            @Override
+            public void process(final JanusGraphVertex vertex, final ScanMetrics metrics) {
+                vertexNum.incrementAndGet();
+            }
+
+            @Override
+            public void getQueries(final QueryContainer queries) {
+                queries.addQuery().properties();
+                queries.addQuery().edges();
+            }
+
+            @Override
+            public VertexScanJob clone() {
+                return this;
+            }
+        });
+
+        Assertions.assertEquals(g.V().count().next(), vertexNum.get());
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unnecessary use of fully qualified name 'Assertions.assertEquals' due to existing static import 'org.junit.jupiter.api.Assertions.assertEquals'](https://app.codacy.com/manual/JanusGraph/janusgraph/pullRequest?prid=5097167)